apps/freeablo/farender/renderer.cpp

```diff
@@ -214,9 +214,9 @@ namespace FARender
             Render::LevelObject obj;
             obj.spriteCacheIndex = object.spriteGroup->getCacheIndex();
             obj.spriteFrame = object.frame;
-            obj.x2 = position.next().x;
-            obj.y2 = position.next().y;
-            obj.dist = position.getDist();

```

Comment:
Maybe we could just pass through Misc::Points here, instead of splitting up the members (I know the old code was messy anyway)